core/src/main/kotlin/com/justai/jaicf/builder/Scenario.kt

```diff
@@ -7,31 +7,37 @@ import com.justai.jaicf.model.scenario.Scenario
 import com.justai.jaicf.model.scenario.ScenarioModel
 import com.justai.jaicf.model.state.State
 import com.justai.jaicf.model.state.StatePath
+import com.justai.jaicf.plugin.StateBody
+import com.justai.jaicf.plugin.StateDeclaration
 import com.justai.jaicf.reactions.Reactions
 
 
 @ScenarioDsl
+@StateDeclaration("")
```

Comment:
Shouldn't we use `name = "/"` here? I know that root is not processed by the plugin, but it'd be better to properly describe scenario structure on jaicf side, for now it looks a little bit confusing

Reply:
I think it's possible